Tests/AFNetworkActivityLoggerTests.m

```diff
@@ -265,5 +265,52 @@ - (void)testThatIndividualLoggerIsNotCalledWhenLoggerIsNilledOut {
     [manager invalidateSessionCancelingTasks:YES];
 }
 
+- (void)testThatResponseSerializerIsAFHTTPResponseSerializerAndResponseBodyIsText {
```

Comment:
add the test case: when _manager.responseSerializer = [AFHTTPResponseSerializer serializer];_ and the http response body is text.